indico/modules/events/editing/client/js/components/TagModal.jsx

```diff
@@ -0,0 +1,95 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Form as FinalForm} from 'react-final-form';
+import {Button, Form, Label, Modal} from 'semantic-ui-react';
+
+import {FinalDropdown, FinalInput, FinalSubmitButton} from 'indico/react/forms';
+import {Translate} from 'indico/react/i18n';
+
+const colors = {
```

Comment:
What about putting this in a `SUIPalette` or `Palette.SUI` object in `indico/web/client/js/utils/palette.js`?

Reply:
Nice, didn't notice this file